client/my-sites/email/email-providers-stacked-comparison/provider-cards/utils.tsx

```diff
@@ -0,0 +1,149 @@
+import formatCurrency from '@automattic/format-currency';
+import { RequestCartProduct } from '@automattic/shopping-cart';
+import { translate } from 'i18n-calypso';
+import page from 'page';
+import React, { FunctionComponent, ReactElement } from 'react';
+import PromoCardPrice from 'calypso/components/promo-section/promo-card/price';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { fillInSingleCartItemAttributes } from 'calypso/lib/cart-values';
+
+export enum TermLength {
+	ANNUALLY = 'annually',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Annually', { context: 'Adverb (as in "Pay annually")'} )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/new-email-comparison-billing-selector)